.github/workflows/RunTests.yaml

```diff
@@ -0,0 +1,158 @@
+name: Run Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: ref on which to run the end-2-end tests (default is head_sha on main)

```

Comment:
NAVContainerHelper's default branch is _master_.

Also, isn't the default the branch on which the workflow is dispatched?

Reply:
fixed